### DISCUSSION TITLE
Decommissioning a Mastodon instance

### DISCUSSION THREAD

**Discussion by isjamesalive** [mastodon/mastodon#22244] [Q&A] [answered]
Are there any procedures that should be followed (in terms of defederating) before permanently deleting a Mastodon instance? Or is it OK to just kill it?

I couldn't find anything in the project documentation or any existing discussions on the subject.

Thanks!

**Comment by wiegelmann** [comment, marked as answer]
https://docs.joinmastodon.org/admin/tootctl/#self-destruct
